codersdk/workspaceapps.go

```diff
@@ -21,6 +21,11 @@ type WorkspaceApp struct {
 	// Icon is a relative path or external URL that specifies
 	// an icon to be displayed in the dashboard.
 	Icon string `json:"icon,omitempty"`
+	// RelativePath denotes whether the app should be accessed via a path on the
+	// `coder server` or via a hostname-based dev URL. If this is set to false
+	// and there is no app wildcard configured on the server, the app will not
+	// be accessible in the UI.
+	RelativePath bool `json:"relative_path"`
```

Comment:
I think we'll have to invert this on the provider to not break current deployments.

This is defaulting to `false`: https://github.com/coder/terraform-provider-coder/blob/main/provider/app.go#L62, which I believe would break everyone.

eg. remove `relative_path` and add `subdomain` instead

Reply:
done!